experimental/split_grpc/client/build.rs

```diff
@@ -23,5 +23,6 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
             &["../../../examples/hello_world/proto/hello_world.proto"],
             &["../../../examples/hello_world/proto/"],
         )?;
+    println!("cargo:rerun-if-changed=../../../examples/hello_world/proto/hello_world.proto");
```

Comment:
Actually the recommended way would be to use `OUT_DIR`, does that not work for us? https://docs.rs/prost-build/0.6.1/prost_build/index.html

Also I think we should reorganise things, moving protos under the folder where they belong (runtime, ABI, SDK), and compiling them once only. Then the compiled artefacts may be re-esported so that they be referenced from other crates, but at least we don't need to deal with proto generation all over the place. WDYT?

cc @daviddrysdale

Reply:
I think this is only a problem because I am compiling the proto files into the source tree. Tonic normally generates the file into the output directory, so this should not generally happen.